custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/write/WriteDataResponse.java

```diff
@@ -40,7 +40,7 @@
 @JsonDeserialize(as = ImmutableWriteDataResponse.class)
 public interface WriteDataResponse {
     /**
-     * Specifies if the operation is successful or not.
+     * Specifies if the operation is successful or not. In case of partial failures, this should be set as false.
```

Comment:
In case of partial failure this flag should be set to false and the error code should be set to PartialWriteFailure.

Reply:
done